example.py

```diff
@@ -0,0 +1,25 @@
+import json
+import requests
+
+def revert_risk_api_request(language, revision_id):
+    use_auth = False
+    inference_url = 'https://api.wikimedia.org/service/lw/inference/v1/models/' + language + '-reverted:predict'
+
+    if use_auth:
+
+        headers = {
+            'Authorization': 'Bearer YOUR_ACCESS_TOKEN',
+            'User-Agent': 'YOUR_APP_NAME (YOUR_EMAIL_OR_CONTACT_PAGE)',
```

Comment:
These would have to be function arguments, otherwise the user won't be able to change them.